### ISSUE TITLE
#Issues

### ISSUE THREAD

**Issue by moosi2020** [LaughingQuoll/TouchBar#6]
Hi I disabled all my tweaks but still no luck using the touch bar 

3D switcher /Flipcontrolcenter
Lockhtml4/miniCC/Noctis
Nodockground/Status peek

**Comment by LaughingQuoll** [comment]
More detail please. Does it not appear?

**Comment by moosi2020** [comment]
Yes it doesn't appear on the home screen or lock screen have reinstalled it twice while tweaks are disabled

**Comment by moosi2020** [comment]
![img_0056](https://cloud.githubusercontent.com/assets/29174420/26761317/a06bdf86-4924-11e7-96ad-bdcb2de45c8d.jpg)
![img_0057](https://cloud.githubusercontent.com/assets/29174420/26761318/a22ec41e-4924-11e7-8120-c2f1a5b7bdfd.jpg)

**Comment by LaughingQuoll** [comment]
What iOS version?

**Comment by moosi2020** [comment]
iphone6 iOS10.1.1

**Comment by moosi2020** [comment]
hi there any advice how to resolve that

**Comment by LaughingQuoll** [comment]
Do you have applist?

**Comment by moosi2020** [comment]
![img_1967](https://user-images.githubusercontent.com/29174420/27168322-19e91d8a-519d-11e7-8262-bc8d585c50f9.JPG)

**Comment by moosi2020** [comment]
much appreciated for you help as i was frustraded i bought iphone7 no jailbreak available 10.3.1

**Comment by moosi2020** [comment]
hi there any update yet
